.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: Continuous Integration
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_PASSWORD: postgres
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      redis:
+        image: redis
+        ports: ['6379:6379']
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - run: bash script/install/config.bash --defaults
+        env:
+          RAILS_ENV: test
+          DATABASE_USERNAME: postgres
+          DATABASE:
+          TEST_DATABASE: nztraintest
+          REDIS_INSTALL: false
+          REDIS_PASS:
+          SCHEDULE_BACKUPS: 0
+          ISOLATE_ROOT: /
+          ISOLATE_CGROUPS: false
+          ISOLATE_BRANCH: master
+
+      - run: AUTOCONFIRM=true script/install.bash --skip-update
+        env:
+          DATABASE_URL: postgresql://postgres:postgres@localhost:5432
+          PGHOST: localhost
+          PGUSER: postgres
+          PGPASSWORD: postgres
+
+      - run: bundle exec rake db:test:load
+      - run: bundle exec rspec
+
+      - uses: coverallsapp/github-action@v1
```

Comment:
Nit: This is using v1, but the current version is v2. Any reason to use v1?

Reply:
Not really, I think it's because I originally wrote this (19b6a8959cd141c4ad5cd490e65605f870e1dbe0) before v2 was fully released. Would be happy to upgrade.